src/reuse/modules/util/formatter.ts

```diff
@@ -156,64 +156,44 @@ export class Formatter {
     let mm: number | string = date.getMonth() + 1;
     const month = date.toLocaleString(locale, { month: "short" });
     const yyyy = date.getFullYear();
-
-    if (sec < 10) {
-      sec = `0${sec}`;
-    }
-
-    if (min < 10) {
-      min = `0${min}`;
-    }
-
-    if (hour < 10) {
-      hour = `0${hour}`;
-    }
-
-    if (dd < 10 && format !== DateFormats.MONTH_DAY_YEAR_COMMA_SHORT) {
-      dd = `0${dd}`;
+    let yy: number | string = yyyy % 100;
+
+    sec = sec.toString().padStart(2, "0");
+    min = min.toString().padStart(2, "0");
+    hour = hour.toString().padStart(2, "0");
+    dd = (dd < 10 && format !== DateFormats.MONTH_DAY_YEAR_COMMA_SHORT) ? dd.toString().padStart(2, "0") : dd;  
```

Comment:
I would move the part of the expression before '?' into a separate local boolean function, `shouldPadDay`. Thus readability could be improved

Reply:
but the code is very different, can you please double check

Reply:
took it from main, as discussed we will refactor it with the suggested code in another PR